dashboard/app/views/scripts/show.html.haml

```diff
@@ -59,7 +59,8 @@
         = "This script is in #{@script.unit_groups.length} course#{'s' unless @script.unit_groups.length == 1}:"
         %ul
           - @script.unit_groups.each do |course|
-            %li= link_to course_path(course)
+            - course_url = course.single_unit_course? ? edit_course_path(course) : course_path(course)
+            %li= link_to course_path(course), course_url
```

Comment:
what do you think about changing it to use the edit_course_path for single-unit courses on this line too, so it's clear the link goes to the edit page?